docs/topics/release-notes-topics/ref_known-issues-8-0-1.adoc

```diff
@@ -13,16 +13,56 @@
 In the {ProductShortName} {WebName}, if you disable the `Allow insecure connection` for an application stored in the SVN repository and edit the application, the {WebName} displays a blank page some times with the `TypeError` `cannot read properties of undefined (reading 'kind')`.
 
 No known workaround exists.  
+
 link:https://issues.redhat.com/browse/MTA-6263[MTA-6263]
 
 .{ProductShortName} assessment form malfunctions when you use it with a custom questionnaire 
 
 In the {ProductShortName} {WebName}, selecting an answer to a question in the assessment form generated from a custom questionnaire causes the form to select an option available for another question.
 
-Workaround: ensure that one or more `order` keys listed in a section, for example `answers`, have a unique order number. link:https://issues.redhat.com/browse/MTA-6262[MTA-6262]
+To work around this problem, ensure that one or more `order` keys listed in a section, for example `answers`, have a unique order number. 
+
+link:https://issues.redhat.com/browse/MTA-6262[MTA-6262]
 
 .{ProductShortName} does not trigger a violation for open source dependencies in Gradle applications 
 
 {ProductShortName} does not trigger a violation for open source libraries in a `source+dependency` analysis of a Gradle application. As a result, the analysis report does not list the incident for open source dependencies. 
 
-No known workaround exists. link:https://issues.redhat.com/browse/MTA-6211[MTA-6211]
\ No newline at end of file
+No known workaround exists.
+
+link:https://issues.redhat.com/browse/MTA-6211[MTA-6211]
+
+
+.{ProductShortName} UI fails to process applications with service bindings
+
+When you use the {ProductFullName} user interface (UI) to run an application discovery for a deployed Cloud Foundry (CF) application with a service binding, {ProductShortName} UI does not parse the `VCAP_SERVICES` environment variables correctly to handle service binding information. Consequently, the discovery fails.
+
+No known workaround exists.
```

Comment:
Cased on this slack thread: [https://redhat-internal.slack.com/archives/C02FNEJ9PLZ/p1763739023626429](https://www.google.com/url?q=https://redhat-internal.slack.com/archives/C02FNEJ9PLZ/p1763739023626429&sa=D&source=docs&ust=1763992175841881&usg=AOvVaw3eWzRTfXhAXlrs57o5tIGJ)

please add that a known workaround is to discover the applications via the CLI